packages/components/pages/f-contact-preferences/src/components/ContactPreferences.vue

```diff
@@ -8,36 +8,41 @@
             has-outline>
             <form @submit.prevent="onFormSubmit">
                 <div
-                    v-for="{ email, key, sms } in preferences"
+                    v-for="{ key, emailEnabled, emailValue, smsEnabled, smsValue } in preferences"
                     :key="key">
-                    <h2 :class="$style['c-contactPreferences-subtitle']">
+                    <h2
+                        :class="$style['c-contactPreferences-subtitle']">
                         {{ $t(`${key}.subtitle`) }}
                     </h2>
-                    <fieldset :class="$style['c-contactPreferences-fieldset']">
+                    <fieldset
+                        :class="$style['c-contactPreferences-fieldset']">
                         <label>
                             <input
```

Comment:
We can probably use f-form-field for this checkbox now. Happy for that to be done in another ticket though.

Reply:
I'm unsure what this entails so if you are cool we can add in the next pr 👍